python/knowhere/__init__.py

```diff
@@ -29,7 +31,7 @@ def CreateIndex(index_name, simd_type="auto"):
         return GPUIVFSQ(-1)
     raise ValueError(
         """ index name only support 
-            'annoy' 'ivf' 'ivfsq' 'hnsw' 'idmap' 'binary_idmap'
+            'annoy' 'ivf' 'bin_ivf' 'ivfsq' 'hnsw' 'idmap' 'binary_idmap'
```

Comment:
should use these names:
annoy, bin_flat, bin_ivf_flat, flat, ivf_flat, ivf_sq8, hnsw